specification/monitor/resource-manager/Microsoft.Insights/stable/2021-09-01/actionGroups_API.json

```diff
@@ -0,0 +1,1120 @@
+        "operationId": "ActionGroups_Delete",
+        "x-ms-examples": {
+          "Delete an action group": {
+            "$ref": "./examples/deleteActionGroup.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ActionGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The action group was successfully deleted."
+          },
+          "204": {
+            "description": "The action group does not exist. It may have already been deleted."
+          },
+          "default": {
+            "description": "An error occurred and the action group could not be deleted.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates an existing action group's tags. To update other fields use the CreateOrUpdate method.",
+        "operationId": "ActionGroups_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ActionGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "actionGroupPatch",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ActionGroupPatchBody"
+            },
+            "description": "Parameters supplied to the operation."
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "An existing action group was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/ActionGroupResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patch an action group": {
+            "$ref": "./examples/patchActionGroup.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/notifications": {
```

Comment:
What is the purpose of this API? As there is no providers information in the API, this API will not be routed to your RP. This API sounds incorrect to me.

Reply:
Hi Ramyasree, I am fairly new to this. Is the provider required? The API is intended to be used without the provider in both of the cases. However if it is required, then I can add it. Could you confirm if the provider is required? Thanks!